pkg/server/plugin/notifier/k8sbundle/k8sbundle_test.go

```diff
@@ -217,10 +225,179 @@ clusters  = [
 	}, test.kubeClient.getConfigMap("NAMESPACE2", "CONFIGMAP2"))
 }
 
-func TestBundleUpdatedWhenCannotCreateClient(t *testing.T) {
-	test := setupTest(t, withKubeClientError())
-	err := test.notifier.NotifyBundleUpdated(context.Background(), commonBundle)
-	spiretest.RequireGRPCStatus(t, err, codes.Internal, "notifier(k8sbundle): failed to create kube clients: kube client not configured")
+func TestBundleInformerAddWebhookEvent(t *testing.T) {
+	plainConfig := `
+webhook_label = "WEBHOOK_LABEL"
+kube_config_file_path = "/some/file/path"
+`
+
+	test := setupTest(t, withPlainConfig(plainConfig))
+	require.NotNil(t, test.rawPlugin.stopCh)
+	test.identityProvider.AppendBundle(testBundle)
+
+	select {
+	case <-test.webhookClient.watcherStarted:
+	case <-time.After(testTimeout):
+		require.FailNow(t, "timed out waiting for watcher to start")
+	}
+
+	webhook := newMutatingWebhook(t, test.webhookClient.Interface, "spire-webhook", "")
+
+	require.Eventually(t, func() bool {
+		actualWebhook, err := test.webhookClient.Get(context.Background(), webhook.Namespace, webhook.Name)
+		require.NoError(t, err)
+		return assert.Equal(t, &admissionv1.MutatingWebhookConfiguration{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:            webhook.Name,
+				ResourceVersion: "1",
+			},
+			Webhooks: []admissionv1.MutatingWebhook{
+				{
+					ClientConfig: admissionv1.WebhookClientConfig{
+						CABundle: []byte(testBundleData),
+					},
+				},
+			},
+		}, actualWebhook)
+	}, testTimeout, time.Second)
```

Comment:
Can we use a small tick interval on all of these `Eventually` assertions? like maybe 50 milliseconds or something? It's the difference between a test suite that takes ~1s to run vs 6s...

Reply:
done